CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+## 0.24.0 / 2018-10-10
+
+* [CHANGE] Default to Node Exporter v0.16.0 (#1812)
+* [CHANGE] Update to Go 1.11 (#1855)
+* [CHANGE] Default to Prometheus v2.4.3 (#1929) (#1983)
+* [CHANGE] Default to Thanos v0.1.0 (#1954)
+* [CHANGE] Overhaul metrics while adding triggerBy metric for Alertmanager (#1984)
+* [FEATURE] Add multi namespace support (#1813)
+* [FEATURE] Add SHA field to Prometheus, Alertmanager and Thanos for images (#1847) (#1854)
+* [FEATURE] Add configuration for priority class to be assigned to Pods (#1875)
+* [FEATURE] Configure sampleLimit per ServiceMonitor (#1895)
+* [FEATURE] Add additionalPeers to Alertmanager (#1878)
+* [FEATURE] Add podTargetLabels to ServiceMonitors (#1880)
+* [FEATURE] Relabel target name for Pods (#1896)
+* [FEATURE] Allow configuration of relabel_configs per ServiceMonitor (#1879)
+* [FEATURE] Add illegal update reconciliation by deleting StatefulSet (#1931)
+* [ENHANCEMENT] Set sane bash options on shell scripts (#1801)
+* [ENHANCEMENT] Update troubleshooting.md (#1808)
```

Comment:
I think so far we have not mentioned any docs updates. So can be skipped as well.